inc/composer/class-docker-compose-generator.php

```diff
@@ -91,7 +91,7 @@ protected function get_php_reusable() : array {
 					'condition' => 'service_started',
 				],
 			],
-			'image' => 'humanmade/altis-local-server-php:4.0.0-dev',
+			'image' => 'humanmade/altis-local-server-php:4.2.0-dev',
```

Comment:
@roborourke this docker image doesn't seem to exist.

Reply:
🤔  strange, I'll look into this

Reply:
I thought it was still pushing 2 versions of the container, looks look Docker hub just has the dev version of the container now. Guess that's fine given we don't pull from there for prod.
```suggestion
			'image' => 'humanmade/altis-local-server-php:4.2.0',
```

Reply:
I also don't see `4.2.0-dev`, but `4.2.0` is [here](https://hub.docker.com/layers/humanmade/altis-local-server-php/4.2.0/images/sha256-3a6efd764fede1b7efc44667b2516dbb33445a39bd6d145cd8baf23a3f137e44?context=explore)